### PR TITLE
[#166672369] forward is_email_enabled field into InitializedProfile

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#166672369](https://www.pivotaltracker.com/story/show/166672369): Come CIT vorrei poter decidere in un unico punto se i messaggi possono essere inoltrati alla mia email oppure no

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/types/__tests__/profile.test.ts [hunk 1]
```diff
@@ -84,6 +85,9 @@ describe("profile type", () => {
     expect(userData.family_name).toBe(mockedUser.family_name);
     expect(userData.fiscal_code).toBe(mockedUser.fiscal_code);
     expect(userData.has_profile).toBeTruthy();
+    expect(userData.is_email_enabled).toBe(
+      fromNullable(mockedExtendedProfile.is_email_enabled).getOrElse(true)
```
Comment (gunzip) [inline src/types/__tests__/profile.test.ts]: `toBe(true)` ? or` toBe(anIsInboxEnabled)` (it makes more clear what's the expected output)

File: src/types/__tests__/profile.test.ts [hunk 2]
```diff
@@ -146,5 +150,10 @@ describe("profile type", () => {
     );
 
     expect(userData.accepted_tos_version).toBe(undefined);
+    expect(userData.is_email_enabled).toBe(
+      fromNullable(mockedExtendedProfileWithoutTos.is_email_enabled).getOrElse(
```
Comment (gunzip) [inline src/types/__tests__/profile.test.ts]: see above

File: src/types/profile.ts [hunk 3]
```diff
@@ -23,6 +24,7 @@ export const toInitializedProfile = (
   family_name: user.family_name,
   fiscal_code: user.fiscal_code,
   has_profile: true,
+  is_email_enabled: fromNullable(profile.is_email_enabled).getOrElse(true),
```
Comment (gunzip) [inline src/types/profile.ts]: is there any reason on why `is_email_enabled` is not required in the specs ?

@cloudify @fpersico 

https://raw.githubusercontent.com/teamdigitale/io-functions-commons/v0.5.0/openapi/definitions.yaml#/ExtendedProfile

File: src/types/__tests__/profile.test.ts (hunk 1 above)
Comment (BurnedMarshal) [reply src/types/__tests__/profile.test.ts]: Fixed in 6df8b41

File: src/types/__tests__/profile.test.ts (hunk 2 above)
Comment (BurnedMarshal) [reply src/types/__tests__/profile.test.ts]: Fixed in 6df8b41

**Pull request merged**